src/contentsharecontroller/DefaultContentShareController.ts

```diff
@@ -105,18 +111,66 @@ export default class DefaultContentShareController
     }
   }
 
-  audioVideoDidStart(): void {
-    this.forEachContentShareObserver(observer => {
-      Maybe.of(observer.contentShareDidStart).map(f => f.bind(observer)());
-    });
-  }
-
   audioVideoDidStop(_sessionStatus: MeetingSessionStatus): void {
     //If the content attendee got dropped or could not connect, stopContentShare will not be called
     //So make sure to clean up the media stream.
     this.mediaStreamBroker.cleanup();
+    if (this.contentShareTile) {
+      this.attendeeAudioVideo.videoTileController.removeVideoTile(this.contentShareTile.id());
+      this.contentShareTile = null;
+    }
     this.forEachContentShareObserver(observer => {
       Maybe.of(observer.contentShareDidStop).map(f => f.bind(observer)());
     });
   }
+
+  private setupContentShareEvents(): void {
+    /*
+     *  We use realtimeSubscribeToAttendeeIdPresence instead of audioVideoDidStart because audioVideoDidStart fires
+     *  before the capacity check in Tincan while when realtimeSubscribeToAttendeeIdPresence fires, we know the
+     *  content attendee has been able to pass the capacity check and join the call so we can start the local
+     *  content share video
+     */
```

Comment:
Prefer single-line `//` over doc-gen style `/* */` for inline comments.

Reply:
Good to know. Fixed.